uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -664,19 +664,20 @@ sub in_range
     return 0 unless $range_string;
     return 0 unless defined($value);
 
-    my @ranges = split(/,/, $range_string);
 
-    my $range = 0;
-    foreach $range (@ranges) {
-       chomp($range);
-       if($range=~/^[0-9.]+$/) { ## single value element
-           return 1 if &floats_are_equal($value, $range, $FLOAT_EQUALS_NB_DECIMALS);
-       } else { ## range X-Y
-           $range =~ /([0-9.]+)-([0-9.]+)/;
-           return 1 if ($1 <= $value && $value <= $2) 
-               || &floats_are_equal($value, $1, $FLOAT_EQUALS_NB_DECIMALS) 
-               || &floats_are_equal($value, $2, $FLOAT_EQUALS_NB_DECIMALS);
-       }
+    if($range_string=~ /,/) {
+        print "Comma separated ranges, as found in $range_string, are no longer supported\n";
```

Comment:
Should it be a print, a print LOG? Should the script exit? Should we also use the notification?

Reply:
This is the perfect place to throw an exception, but since we do not have access to them (yet...), I would go with "die". Maybe also put a "TODO" that says that this should be refactored to throw an exception when the Moose PR is merged.

Reply:
Usually I would not be in favor of a function call "dying" on us but in this specific case I think it makes sense.